### DISCUSSION TITLE
node-chat-api似乎失效了，无论发送什么都报错：The moderation filter triggered. Try again with different wording.

### DISCUSSION THREAD

**Discussion by OUYANGHW** [viopsa233/AI-testers-Forum#21] [Q&A] [answered]
![1682387539832](https://user-images.githubusercontent.com/42854960/234155089-8af91174-0839-4f73-acd6-bdff9f30c107.jpg)

**Comment by ghost** [comment]
果然还是来了，今天早上Edge侧边栏和Skype也失效

**Comment by liukaixiang817** [comment]
提示词也完全失效了

**Reply by liukaixiang817** [reply]
但是好像之前那个python的脚本如果用edge直接打开的话边栏还能用，只是无法保存聊天记录了

**Comment by araea** [comment, marked as answer]
没失效，早上我用cli和Pandora都没有什么问题，现在我手机上也没问题，建议重新去原项目
页面下载更新一下版本。

![Screenshot_2023-04-25-11-11-16-151_com.termux.jpg](https://user-images.githubusercontent.com/120614554/234165217-04df0687-8236-45d4-9263-28630b01a8e6.jpg)

**Reply by ghost** [reply]
https://github.com/waylaidwanderer/node-chatgpt-api/issues/330
已经有人提issue了，可能微软还在灰度测试所以你的没问题

**Reply by araea** [reply]
刚刚试了一下，指令还能用？
![Screenshot_2023-04-25-11-38-38-252_com.microsoft.bing.jpg](https://user-images.githubusercontent.com/120614554/234169167-7969a3b9-4123-4be7-8835-b3d633bb8f5c.jpg)

**Reply by OUYANGHW** [reply]
非常奇怪，我把本地项目删除并重新下载源代码居然就可以了，可能是我修改了代码导致？

**Reply by araea** [reply]
> 非常奇怪，我把本地项目删除并重新下载源代码居然就可以了，可能是我修改了代码导致？

其实作者一直在更新的，只是没写出来，你看那个更新日期，其实最近有过修改。

**Comment by vifdiwxfs** [comment]
同样，用了很久。突然没了这么方便的工具，很不习惯了。

**Reply by OUYANGHW** [reply]
我重新下载了源代码，这解决了我的问题。

**Reply by trz98** [reply]
我也是在重新部署项目后恢复正常，感谢

**Reply by trz98** [reply]
https://github.com/waylaidwanderer/node-chatgpt-api/blob/d2edb8122006c55bb3fd5ef79ef0f5b70c2cb9a8/src/BingAIClient.js#L325我的错误原因是因为修改了这行代码

**Reply by vifdiwxfs** [reply]
重新部署后回复正常了😃

**Comment by jishux2** [comment]
我按 https://github.com/InterestingDarkness/node-chatgpt-api/commit/412f4691e4596a98f96bd2cd2e9639e61e3017b9 改了代码，这样做输入信息会触发审查机制，有时没事，有时就莫名其妙提示The moderation filter triggered
